edgexfoundry/device-rfid-llrp-go
# refactor: Consume Provision Watcher changes for running multiple instances

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-rfid-llrp-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
Run non-secure EdgeX Stack
Build service from this PR branch
run service with `-cp -d -o` flags
Verify service starts w/o errors and logs have the following
```
"ProvisionWatcher llrp-impinj-provision-watcher not found in Metadata, adding it..."
"ProvisionWatcher llrp-provision-watcher not found in Metadata, adding it..."
"provision watcher llrp-impinj-provision-watcher added"
"provision watcher llrp-provision-watcher added"
```
Restart the service with the `-cp -d -o -i 1` flags
Verify service starts w/o errors and logs have the following
```
"ProvisionWatcher llrp-impinj-provision-watcher exists, using the existing one"
"ProvisionWatcher llrp-provision-watcher exists, using the existing one"
```

## New Dependency Instructions (If applicable)
